foundation_cms/base/models/abstract_base_page.py

```diff
@@ -9,10 +9,22 @@
 from wagtail.snippets.models import register_snippet
 from wagtailmetadata.models import MetadataPageMixin
 
-from foundation_cms.blocks import TabbedContentContainerBlock, TwoColumnContainerBlock
+from foundation_cms.blocks import TabbedContentContainerBlock, TwoColumnContainerBlock, ImpactNumberBlock
 from foundation_cms.blocks.image_block import CustomImageBlock
 from foundation_cms.blocks.audio_block import AudioBlock
 
+# Shared StreamField block types for use across pages that inherit from AbstractBasePage.
+# Extend this list in specific page models (e.g., HomePage) to add more blocks as needed.
+base_page_block_options = [
+    ("rich_text", RichTextBlock()),
+    ("image", CustomImageBlock()),
+    ("audio", AudioBlock()),
+    ('tabbed_content', TabbedContentContainerBlock()),
+    ("two_column_container_block", TwoColumnContainerBlock()),
+    ("impact_numbers", ImpactNumberBlock()),
+]
```

Comment:
Not directly related to this PR, but I thought it would be helpful for the future!